content/solana-nft.md

```diff
@@ -0,0 +1,637 @@
+    "solTreasuryAccount": "<YOUR_WALLET_ADDRESS>",
+    "splTokenAccount": null,
+    "splToken": null,
+    "goLiveDate": "25 Dec 2020 00:00:00 GMT",
+    "endSettings": null,
+    "whitelistMintSettings": null,
+    "hiddenSettings": null,
+    "storage": "arweave",
+    "ipfsInfuraProjectId": null,
+    "ipfsInfuraSecret": null,
+    "awsS3Bucket": null,
+    "nftStorageKey": null,
+    "noRetainAuthority": false,
+    "noMutable": false
+}
+```
+
+Update the `solTreasuryAccount` field with the wallet address we imported to Phantom. This will be the only setting we change for now.
+
+```json
+    "solTreasuryAccount": "<YOUR_WALLET_ADDRESS>"
+```
+
+### 5. Prepare Assets
+
+Next, let's prepare the assets for our Candy Machine. The starter code includes an `assets` folder that has the images and the metadata files we'll be using for our collection.
+
+Open each `.json` file and update the `address` field with the wallet address we imported to Phantom.
+
+```json
+{
+    "name": "1",
+    "symbol": "TEST",
+    "image": "0.png",
+    "properties": {
+        "files": [{ "uri": "0.png", "type": "image/png" }],
+        "category": "image",
+        "creators": [
+            {
+                "address": "<YOUR_WALLET_ADDRESS>",
+                "share": 100
+            }
+        ]
+    },
+    "description": "",
+    "seller_fee_basis_points": 500,
+    "attributes": [
+        { "trait_type": "background", "value": "blue" },
+        { "trait_type": "eyes", "value": "star-eyes" },
+        { "trait_type": "mouth", "value": "triangle-mouth" },
+        { "trait_type": "face", "value": "teal-face" }
+    ],
+    "collection": { "name": "NAME", "family": "FAMILY" }
+}
+```
+
+Once you’ve updated the `address` field for each `.json` file, let's verify that the assets are ready for upload by running the `verify_assets` command.
+
+This will check that:
+
+1. Files types are supported
+2. Each image has a corresponding `.json` metadata file with the correct index naming
+3. The `creators` field is consistent across all `.json` files
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts verify_assets ./assets
+```
+
+The output should look something like this:
+
+![Gif of Metaplex Verify Assets](../assets/solana-nft-metaplex-verify-assets.gif)
+
+### 6. Create Candy Machine
+
+Now that we’ve configured our Candy Machine and verified that our assets are ready for upload, let's upload our assets and create our Candy Machine by running the `upload` command.
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts upload \
+    -e devnet \
+    -k ~/.config/solana/devnet.json \
+    -cp config.json \
+    -c example \
+    ./assets
+```
+
+The output should look something like this:
+
+![Gif of Metaplex Upload](../assets/solana-nft-metaplex-upload.gif)
+
+There will now be a `.cache` folder with a `devnet-example.json` file that includes the address of our Candy Machine and the uploaded Arweave link for each NFT.
```

Comment:
I feel like we need a bit more explanation of Arweave in the Overview and maybe in the demo somewhere too

Reply:
updated